pages.pt_BR/common/ebook-convert.md

```diff
@@ -0,0 +1,13 @@
+# ebook-convert
+
+> Pode ser usado para converter e-books entre os fomatos comuns, como PDF, EPUB e MOBI.
+> Faz parte da biblioteca de ferramentas *Calibre e-book*.
+> Mais informações: <https://manual.calibre-ebook.com/generated/en/ebook-convert.html>.
+
+- Converter um e-book em outro formato:
+
+`ebook-convert {{original}} {{formato_pretendido}}`
```

Comment:
This is not clear in the main page. Could you use `path/to/input_file` and `output_file` in the English version?
```suggestion
`ebook-convert {{caminho/para/arquivo_entrada}} {{arquivo_saída}}`
```

Reply:
I follow the instructions for translated content in this template: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md

Reply:
It look better in your way!